src/oc_erchef/apps/oc_chef_wm/itest/mocks/chef_otto.erl

```diff
@@ -6,12 +6,9 @@
 -module(chef_otto).
 
 -export([
-         connect/0,
-         fetch_org_id/2
+         connect/0
```

Comment:
I searched mover for this and other modified functions and didn't find any references to it - either at chef_db or chef_otto level.